rep/processors/CtGovPdfProcessor.py

```diff
@@ -35,7 +46,9 @@ def _process_blob_wrapper(self, blob):
             t2 = "".join(re.findall(self.UNKOWN_PATTERN, votes[i][1])).strip()
```

Comment:
same here for the variable names `t1`, `t2`. I didn't rename them when I ported this code because I had no idea what they represented

Reply:
👍